spec/services/middleware_topology_service_spec.rb

```diff
@@ -17,7 +17,7 @@
     subject { middleware_topology_service.build_topology }
 
     let(:ems_hawkular) { FactoryGirl.create(:ems_hawkular) }
-    let(:middleware_server) do
+    let(:md_server) do
```

Comment:
I wouldn't know that `md_server` is middleware_server.  do we need server at all?  Can we call it `middleware`?